docs/getting-started.md

```diff
@@ -77,6 +77,19 @@ Verder zijn er nog een paar algemene functies die gelden voor alle bovenstaande
 - Bij het zoeken van een pand op **locatie** moet de header **Content-Crs** worden meegestuurd. De API ondersteunt op dit moment alleen het RD coördinatenstelsel (epsg:28992).
```

Comment:
we hebben hier ook zoeken op bbox aan toegevoegd, zowel voor panden als voor adresseerbare objecten

Reply:
De content-crs header is niet meer verplicht. Wanneer je deze weglaat wordt default coördinatenstelsel RD (epsg:28992) gebruikt.

Reply:
Wijzigingen aangebracht n.a.v. je opmerkingen.